### PR TITLE
New SMPS site

### PR DESCRIPTION
**Pull request by chandrashritii**
**Description**

Website for the SMPS project

**Notes for Reviewers**

Site Layout:

- What is SMPS?
- Partners
- Resources
- SMPS Live
- Blog [To be added]
- Connect 

Fixes Issue #45

### CODE REVIEW

File: docs/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
Comment (NupurThakur27) [inline docs/LICENSE.txt:1]: Why the name of the file LICENSE.txt and not just LICENSE

File: docs/.github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+name: Bug report
+about: Create a report to help us improve
```
Comment (kushthedude) [inline docs/.github/ISSUE_TEMPLATE/bug_report.md:3]: Remove the .github folder

File: docs/LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+Copyright (c) 2019 Ravi Riley
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
```
Comment (kushthedude) [inline docs/LICENSE.txt:7]: Remove this

**Comment by leecalcote** [comment]
No small amount work completed here. 😅

File: docs/LICENSE.txt (hunk 1 above)
Comment (chandrashritii) [reply docs/LICENSE.txt:1]: @NupurThakur27 Thanks for pointing that out! It needs to be updated/removed. I'll get it done :)

**Pull request merged**